tianshou/env/venvs.py

```diff
@@ -383,11 +388,29 @@ class SubprocVectorEnv(BaseVectorEnv):
         Please refer to :class:`~tianshou.env.BaseVectorEnv` for other APIs' usage.
     """
 
-    def __init__(self, env_fns: Sequence[Callable[[], ENV_TYPE]], **kwargs: Any) -> None:
+    def __init__(
+        self,
+        env_fns: Sequence[Callable[[], ENV_TYPE]],
+        wait_num: int | None = None,
+        timeout: float | None = None,
+        share_memory: bool = False,
+        context: Literal["fork", "spawn"] | None = None,
+    ) -> None:
+        """:param share_memory: whether to share memory between the main process and the worker process. Allows for
+        shared buffers to exchange observations
+        :param context: the context to use for multiprocessing. Consider using "fork" when using macOS and additional
+         parallelization through, for example, joblib. Defaults to None, which will use the default system context.
```

Comment:
not indented correctly (here and above), probably that's why the CI pipeline fails